qrcode/main.py

```diff
@@ -76,7 +89,10 @@ def make(self, fit=True):
         """
         if fit or (self.version is None):
             self.best_fit(start=self.version)
-        self.makeImpl(False, self.best_mask_pattern())
+        if self.mask_pattern == None:
```

Comment:
Right. Now I learn that it's better to use `is` [when you want to check against an object's identity](https://stackoverflow.com/questions/14247373/python-none-comparison-should-i-use-is-or).
self.mask_pattern can be 0 so the latter should changed to `self.mask_pattern if self.mask_pattern is None else self.best_mask_pattern()`. And that may exceed 80 characters in one line.